content/en/synthetics/ci_results_explorer.md

```diff
@@ -8,7 +8,7 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/datadog-synthetic-ci-cd-testing/"
   tag: "Blog"
   text: "Incorporate Datadog Synthetic tests into your CI/CD pipeline"
-- link: "https://learn.datadoghq.com/course/view.php?id=37"
+- link: "https://learn.datadoghq.com/enrol/index.php?id=37"
```

Comment:
Not sure if this is meant to be with one L (as in, the url itself is naturally misspelled), but flagging just in case

Reply:
The old link redirects to this new link with "enrol," good eye!